README.md

```
@@ -94,11 +94,21 @@ sudo pacman -S --needed base-devel ncurses
+**FreeBSD/Termux**
+~~~ shell
+pkg install ncurses automake autoconf clang
+~~~
+
 ### Compile from source:
 To compile from source, download from the Git repository (`git clone` or downloads from [GitHub releases](https://github.com/htop-dev/htop/releases/)), then run:
 ~~~ shell
 ./autogen.sh && ./configure && make
 ~~~
+then, run:
+~~~ shell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)